inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/listener/StartupSortListener.java

```diff
@@ -64,7 +64,7 @@ public ListenerResult listen(WorkflowContext context) throws Exception {
         String dataFlows = kvConf.get(InlongGroupSettings.DATA_FLOW);
         String inlongGroupId = context.getProcessForm().getInlongGroupId();
         if (StringUtils.isEmpty(dataFlows)) {
-            String message = String.format("inlongGroupId:%s not exec startupProcess listener,dataflows is empty",
+            String message = String.format("inlongGroupId:%s not exec startupProcess listener, dataflows is empty",
```

Comment:
Change to `groupId [%s] not exec startupProcess listener, as the dataflows is empty`.

Reply:
done